src/Controls/tests/TestCases.Shared.Tests/UITest.cs

```diff
@@ -63,7 +63,7 @@ public override IConfig GetTestConfig()
 				case TestDevice.iOS:
 					config.SetProperty("DeviceName", Environment.GetEnvironmentVariable("DEVICE_NAME") ?? "iPhone Xs");
 					config.SetProperty("PlatformVersion", Environment.GetEnvironmentVariable("PLATFORM_VERSION") ?? "18.0");
-					config.SetProperty("Udid", Environment.GetEnvironmentVariable("DEVICE_UDID") ?? "");
+					config.SetProperty("Udid", Environment.GetEnvironmentVariable("DEVICE_UDID") ?? "FE996E6B-0D88-4D9B-A7BE-72AB288FCA9E");
```

Comment:
I think this changes is a mistake. Probably added a specific Udid testing. Could you revert it?

Reply:
@jsuarezruiz , I have reverted the changes. please let me know if you have any concern.